Citavi 5 XML.js

```diff
@@ -247,10 +247,19 @@ function doImport() {
 				} else if (addressType == "PubMedId" && item.extra.indexOf("PMID") == -1) {
 					addExtraLine(item, "PMID", address);
 				} else {
-					item.attachments.push({
-						url: address,
-						title:"Location"
-					});
+					// distinguish between local paths and internet addresses
+					// (maybe also encoded in AddressInfo subfield?)
+					if (address.indexOf('http://')==0 || address.indexOf('https://')==0) {
+						item.attachments.push({
+							url: address,
+							title:"Location"
+						});
+					} else {
+						item.attachments.push({
+							path: address,
+							title: address
```

Comment:
shouldn't this be something like "Full Text"? the filepath as title seems odd to me.

Reply:
It would only be the name of the file (relative path). But "Full Text" is more in line with other translators (also it can be any PDF attached to an item, e.g. also a TOC for a book).